Gemfile

```diff
@@ -96,7 +97,6 @@ gem "timecop", group: %i[test cucumber] # date/time manipulation in tests
 group :test, :development do
   # gem "debase"
   gem "decko-spring", path: "./vendor/decko/support"
-  gem "puma", "~>5.6"                   # local webserver (6.x broke semaphore )
```

Comment:
Is the notion that this is an interim solution and that we'll move to something more powerful before going live? Puma is a pretty lightweight webserver. I'd have to look a little more closely at our apache configurations to make specific arguments about what it can and can't do. 

But maybe the idea is that since each container has its own webserver, ligthweight is best?